### PR TITLE
v2.0.1 Quality of Live Improvements

### PR DESCRIPTION
**Pull request by sauterl**
Adds various QoL improvements:

- Re-adds the timestamp in the submission on an evaluation run export (thanks @lucaro )
- Adds a filter to the team building dialog (such that those user lists can be filtered) and auto-scrolling with the dragable
- Added some robustness to the recovery of active runs: If for some reason a run is active which does not have any tasks or teams, the run is ended on startup
- The create-run button is disabled if the template has no tasks or no teams
- Minor Api type streamlining: ApiTarget, ApiHint and ApiAnswer all hold a reference to the media item now (in the property `item`), if they are of a type with an item.
- Updated Open Api Sepcifications accordingly
- Minor bugfixes

Bumps version to 2.0.1

### CODE REVIEW

File: doc/oas-client.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "openapi" : "3.0.3",
   "info" : {
     "title" : "DRES Client API",
-    "version" : "2.0.0-RC5",
-    "description" : "Client API for DRES (Distributed Retrieval Evaluation Server), Version 2.0.0-RC5"
+    "version" : "2.0.1-SNAPSHOT",
```
Comment (lucaro) [inline doc/oas-client.json:5]: Why is this a snapshot?

File: doc/oas.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "openapi" : "3.0.3",
   "info" : {
     "title" : "DRES API",
-    "version" : "2.0.0-RC5",
-    "description" : "API for DRES (Distributed Retrieval Evaluation Server), Version 2.0.0-RC5",
+    "version" : "2.0.1-SNAPSHOT",
```
Comment (lucaro) [inline doc/oas.json:5]: same here?

File: frontend/src/app/template/template-builder/components/team-builder-dialog/team-builder-dialog.component.ts [hunk 3]
```diff
@@ -163,6 +182,16 @@ export class TeamBuilderDialogComponent {
     } as ApiTeam;
   }
 
+  onMemberFilterChanged(filter: string){
+    console.log(`MEMBER: was: ${this.memberFilterText} is: ${filter}`)
+    this.memberFilterText = filter;
```
Comment (lucaro) [inline frontend/src/app/template/template-builder/components/team-builder-dialog/team-builder-dialog.component.ts:187]: do you want to keep this or was this for debugging?

**Review by lucaro — CHANGES_REQUESTED** [review]

File: doc/oas-client.json (hunk 1 above)
Comment (sauterl) [reply doc/oas-client.json]: Done

File: doc/oas.json (hunk 2 above)
Comment (sauterl) [reply doc/oas.json]: Done

File: frontend/src/app/template/template-builder/components/team-builder-dialog/team-builder-dialog.component.ts (hunk 3 above)
Comment (sauterl) [reply frontend/src/app/template/template-builder/components/team-builder-dialog/team-builder-dialog.component.ts:186]: Thanks, this was for debugging and has been removed.

**Review by lucaro — APPROVED** [review]

**Pull request merged**